### ISSUE TITLE
Feature: global stagger_phase_limit

### ISSUE THREAD

**Issue by jneuhauser** [ericaltendorf/plotman#551]
**Describe the request**
Since phase 1 uses multiple threads, it would be good to also define a global stagger_phase_limit so that we can set a global phase limit for all plots on this machine regardless of the temple drive.

For sure i can use global_stagger_m with an approximate time for phase 1 on my machine, but this needs manual intervention on every machine.

I don't know if this is easily possible with the current code...

**Comment by jneuhauser** [comment]
I've seen a related PR about thie here: https://github.com/ericaltendorf/plotman/pull/514

**Comment by ShocWave** [comment]
Yes, we need this. I'm plotting using multiple 7200rpm drives drives for the temp dirs, and only want one job to be in phase one at a time. However, tmpdir_stagger_phase_major does not respect this across multiple temp dirs, and I end up with all my jobs in phase 1, slowing the computer to the point where I can barely even move the mouse.

**Comment by jneuhauser** [comment]
I'm using this since some days for the same purpose as you mentioned...

You can install latest development version with the applied PR by:
`pip install --force-reinstall git+https://github.com/ericaltendorf/plotman@refs/pull/514/merge`

**Comment by altendky** [comment]
The tmp dir stagger stuff is per drive.  What is it about phase 1 that makes you want to limit the jobs there?  The multiple threads that you can configure in that phase?
